patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -181,7 +181,15 @@
        func_196254_a("dropper", new BlockDropper(Block.Builder.func_200945_a(Material.field_151576_e).func_200943_b(3.5F)));
        func_196254_a("white_terracotta", new Block(Block.Builder.func_200949_a(Material.field_151576_e, MapColor.field_193561_M).func_200948_a(1.25F, 4.2F)));
        func_196254_a("orange_terracotta", new Block(Block.Builder.func_200949_a(Material.field_151576_e, MapColor.field_193562_N).func_200948_a(1.25F, 4.2F)));
-@@ -1604,4 +1620,83 @@
+@@ -1464,6 +1480,7 @@
+       func_196254_a("structure_block", new BlockStructure(Block.Builder.func_200949_a(Material.field_151573_f, MapColor.field_197656_x).func_200948_a(-1.0F, 3600000.0F)));
+       field_149771_c.func_177776_a();
+ 
++      if(false) // Processed in GameData.BlockCallbacks#onBake
```

Comment:
The previous patch here deleted this block rather than just skipping over it, if that was an intentional decision then this should probably do the same.

Reply:
Ya there are times where we use the patches as documentation of deleted/moved sections.